docs/modules/ROOT/partials/storage-ceph-remove-mon.adoc

```diff
@@ -0,0 +1,107 @@
+
+. Find the MON (if any) on the node to be replaced
++
+[source,bash,subs="attributes+"]
+----
+MON_ID=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods \
+  -lapp=rook-ceph-mon -o wide \
+  | grep "{mon_node}" | cut -d- -f4)
+echo $MON_ID
+----
++
+TIP: You can skip the remaining steps in this section if `$MON_ID` is empty.
+
+ifeval::["{argocd_disabled}" != "true"]
```

Comment:
Might be better to use strings "yes" and "no" to make it clear that we're doing string compares